Entities/Actor.h

```diff
@@ -1351,9 +1347,9 @@ ClassInfoGetters
     Vector m_ViewPoint;
     // The inventory of carried MovableObjects of this Actor. They are also Owned by this.
     std::deque<MovableObject *> m_Inventory;
-    // The max recommended weight of this Actor, including himself and all his inventory
-    // If this is 0 or less, there's no recommended limit
-    float m_MaxMass;
+    // The max combined weight of items in this Actor's inventory.
+    // If this is -1 there's no limit.
+    float m_MaxInventoryMass;
```

Comment:
Might as well redo this member variable comment in our modern standard, and tweak it to be a little better
`float m_MaxInventoryMass; //!< The mass limit for this Actor's inventory. -1 means there's no limit.`